### PR TITLE
Add commands for MacOS and fixed typos.

### PR DESCRIPTION
**Pull request by afrancht**
- Added commands for MacOS
- Fixed some typos

### CODE REVIEW

File: docs/source/guides/staking_guide.rst [hunk 1]
```diff
@@ -114,13 +114,22 @@ off).
 
 If you don't have a hardware wallet, you can create a software one:
 
+Whilst running the initialized node:
```
Comment (cygnusv) [inline docs/source/guides/staking_guide.rst]: Maybe put this line together with the previous one:

"If you don't have a hardware wallet, you can create a software one. Whilst running the initialized node:"

File: docs/source/guides/staking_guide.rst [hunk 2]
```diff
@@ -215,8 +226,8 @@ attached a Worker node to your Staker, so you still have to do it!
 Bond an Ursula to a Staker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After you create an Ethereum node for your worker (see below about the worker),
-you can set the worker. Stakes which don't yet have a worker will be
+After you create an Ethereum node for your staker (see below about the worker),
```
Comment (mswilkison) [inline docs/source/guides/staking_guide.rst]: I think it's actually correct to say the worker has the Ethereum node, right?

Comment (cygnusv) [reply docs/source/guides/staking_guide.rst]: YEs, but this step is referred to the Staker, since she is setting now her worker.

Comment (KPrasch) [reply docs/source/guides/staking_guide.rst]: It's subtle - but the original intention is to indicate that the staker must know about the worker's address to call `set-worker`.

**Pull request merged**